src/components/organisms/FridgeList/FridgeList.jsx

```diff
@@ -0,0 +1,122 @@
+import { Button, List, ListItem, Stack, Typography } from '@mui/material';
+import {
+  CalendarMonthOutlined as CalendarIcon,
+  Instagram as InstagramIcon,
+  LocationOnOutlined as LocationOnOutlinedIcon,
+} from '@mui/icons-material';
+
+import Image from 'next/image';
+import Link from 'next/link';
+import PropTypes from 'prop-types';
+import { getInstagramHandleFromUrl } from 'lib/helperFunctions';
+import typesValidation from 'schema/api/fridge/prop-types';
+import { useMemo } from 'react';
+
+function indexReportsByFridgeId(reports) {
+  const ret = {};
+  for (const report of reports) {
+    ret[report.fridgeId] = report;
+  }
+  return ret;
+}
+
+function Location({ location }) {
+  return (
+    <Stack direction="row" spacing={3} alignItems="center">
+      <LocationOnOutlinedIcon />
+      <Typography sx={{ fontSize: ['0.9375rem'], color: 'text.primary' }}>
+        {`${location.street} ${location.city}, ${location.state} ${location.zip}`}
+      </Typography>
+    </Stack>
+  );
+}
+Location.propTypes = {
+  location: PropTypes.object.isRequired,
+};
+
+function Instagram({ instagramUrl }) {
+  const handle = getInstagramHandleFromUrl(instagramUrl);
+  return (
+    <Stack direction="row" spacing={3} alignItems="center">
+      <InstagramIcon />
+      <Typography sx={{ fontSize: ['0.9375rem'], color: 'text.primary' }}>
+        @{handle}
+      </Typography>
+    </Stack>
+  );
+}
+Instagram.propTypes = {
+  instagramUrl: PropTypes.string.isRequired,
+};
+
+function LastUpdate({ date }) {
+  return (
+    <Stack direction="row" spacing={3} alignItems="center">
+      <CalendarIcon />
+      <Typography sx={{ fontSize: ['0.9375rem'], color: 'text.primary' }}>
+        Last Update: {date}
```

Comment:
How do we want to format the date here?
![image](https://user-images.githubusercontent.com/8978815/188774216-3f545de3-3896-4978-86cc-d03235b75006.png)

Reply:
Lets convert to local timezone and then display it in one of these formats

I was initially thinking one of these formats but it can get confusing 
03/29/2022 01:19 PM
2022-03-29 01:19 PM

So how about something like this?
MM DD, YYYY TIME
January 29, 2022 01:19 PM

Reply:
I'm moving the FridgeList component into the Information organism. There are existing functions to format all the fields in the Fridge and Report records. They are in the ./components file.

Reply:
Fixed by using `formatDate` function from `lib/helperFunctions`.
de097e73f1f091cc4a5c8348e5a4f51475ac56b6